Organization.md

```diff
@@ -0,0 +1,102 @@
+# Organization
+
+Currently, the Distributed Text Services is not legally formalized as an organization (outside of the Github Organization and the Google Group).
+
+Although, the Distributed Text Services specifications is lead by a [Technical Committee](#technical_committee). It is responsible for the evolution and documentation of the specifications.
+
+
+## Technical Committee
+
+* The **Technical committee** is responsible for voting on feature requests and bugs
+* Once the initial committee is established, the committee itself is responsible for approving new members to the committee
+*  New feature requests must be voted on by the technical committee in order to be approved as additions to the spec.
+* Committee members have 1 month after a feature request is officially opened for voting to provide their vote.  If you don't vote within that time frame, it is the same as voting yes. The exception is feature requests opened for voting in July and August. The voting period for these is extended to 2 months.
```

Comment:
change "If you don't vote within that time frame" to "Not submitting a vote within that time frame"

Reply:
change "The exception is" to "An exception is made for"